### PR TITLE
Documentation update

### PR DESCRIPTION
**Pull request by cmm79**
#### General

- All data issues and labels imported from 2023 edition
- Need confirmation: Can I delete all the branches that come from the 2023 repo and leave only `main` and the this branch I'm working on?
- Created and assigned milestones to all issues
- Updated and created Tracking issues for all SLUs

#### `md` Files:

- Many updates, especially the WSL 2 part
I need someone to check Mac and based Linux system procedures because I don't have does systems...

**requirements.txt**

 - Reviewed to the smallest list we could get to (8 'items')

### CODE REVIEW

**Review by kagglekim — APPROVED** [review]
Looks great!

File: troubleshooting.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
-[6. When setting up python 3.8 I get an error](#6-when-setting-up-python-38-i-get-an-error)   
-[7. Nothing happens when I type my password](#7-nothing-happens-when-i-type-my-password)   
-[8. I still have a NotImplemented error](#8-i-still-have-a-notimplemented-error)   
+[1. When I open Windows Explorer through Ubuntu, it goes to a different folder than in the guide](#1-when-i-open-windows-explorer-through-ubuntu-it-goes-to-a-different-folder-than-in-the-guide)  
+[2. Ubuntu on Windows 10 high CPU usage, crashes](#2-ubuntu-on-windows-10-high-cpu-usage-crashes)  
+[3. When I pull from the `ds prep course-2024` repository, I get an error](#3-when-i-pull-from-the-ds-prep-course-2024-repository-i-get-an-error)  
+[4. When I try to open 'jupyter notebook', I get an error](#4-when-i-try-to-open-the-jupyter-notebook-i-get-an-error)  
+[5. When I use the `cp` command the `>` sign appears and the command does not execute](#5-when-i-use-the-cp-command-the--sign-appears-and-the-command-does-not-execute)  
+[6. When setting up python 3.8 I get an error](#6-when-setting-up-python-38-i-get-an-error)
```
Comment (Gustavo-SF) [inline troubleshooting.md]: Should it be python 3.11 now?

**Review by Gustavo-SF — APPROVED** [review]
LGTM

**Comment by Gustavo-SF** [comment]
Yes you can delete all the branches.

File: troubleshooting.md (hunk 1 above)
Comment (cmm79) [reply troubleshooting.md]: Yes, you're right. Missed that one :(
Thanks!
Done

**Review by majkah0 — APPROVED** [review]

**Pull request merged**